src/word2vec.c

```diff
@@ -553,8 +585,23 @@ void TrainModel() {
   InitNet();
   if (negative > 0) InitUnigramTable();
   start = clock();
+  
+#ifdef _MSC_VER
+	HANDLE *pt = (HANDLE *)malloc(num_threads * sizeof(HANDLE));
+	for (int i = 0; i < num_threads; i++){
+		pt[i] = (HANDLE)_beginthreadex(NULL, 0, TrainModelThread_win, (void *)i, 0, NULL);
+	}
+	WaitForMultipleObjects(num_threads, pt, TRUE, INFINITE);
+	for (int i = 0; i < num_threads; i++){
+		CloseHandle(pt[i]);
+	}
+	free(pt);
```

Comment:
This `free` should actually be behind the `#endif` below. Otherwise the memory leaks on linux.